### PR TITLE
Airport challenge

### PR DESCRIPTION
**Pull request by roryai**

### CODE REVIEW

**Comment by roryai** [comment]
I'm running Ruby 2.3.1, did challenge using that version, and changed gemfile to 2.3.1 before I started. Travis still failing. Seems like RVM is mentioned but I'm using rbenv.

File: RORY_README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+SYNTAX - How do I check whether a class method stores the parameter passed to it as an instance variable?
+
+QUESTIONS ---------------
+
+How do I remove a specific plane from an array? Pop only useful if it's on the end. This may be useful but haven't time to test it now: airport::planes.select { |a| a == self}
+SOLVED ABOVE - used Array.delete
+
+Why does my rspec look so different? Coveralls doesn't seem to have loaded properly. Missing my .....F.. at the top :(
+
+  Why does my Plane flying status fail if I put a ? on the end? rspec tells me "syntax error, unexpected '='". Had to remove ?
```
Comment (maryrosecook) [inline RORY_README.md]: Happy to talk about these questions! Way to go for writing this document.  Really good reflections!

File: lib/airport.rb [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  def takeoff(plane)
+  #  fail 'Cannot take-off, too stormy' if stormy?
+    @planes.delete(plane)
+    plane::flying = true
+  end
+
+  def stormy?
+    heavens = ['fine', 'fine', 'fine', 'stormy']
+    return true if heavens.sample == 'stormy'
+    return 'fine'
```
Comment (maryrosecook) [inline lib/airport.rb]: These two lines feel hard to understand.

File: RORY_README.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+I want to write a test that checks whether the parameter I pass to a class method is assigned to an instance variable. I have no idea how to write this. I have spent a long time wondering whether it's even possible to test for this and asking myself what the point of the test would be as it's blindingly obvious whether it does this or not from looking at the code. I'm leaning towards concluding that rspec is good for testing more complex features but that for the simplest elements of a program it's best to create them without testing them.
+
+9pm Sunday night - Decided to write the program and then write rspec tests afterwards. I am aware that this goes against the practice and purpose of TDD and decided to do it because my rspec syntax ability is very far behind where I want it to be and I will learn the syntax more quickly if I can isolate the thought process of writing the test from having to work out what the program needs to look like to pass it. My problem is not the process of TDD, but stumbling over rspec syntax due to my current low level rspec vocabulary. I am aware of the benefits and system of TDD.
+
```
Comment (maryrosecook) [inline RORY_README.md]: Understand the reasons here.  It's certainly worth being ruthless about being test first.  It's helps with your architecture as well as your code correctness.  Perhaps practice rspec against some dummy application code?

**Pull request closed without merging**